<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PngyuMainWindow</class>
 <widget class="QMainWindow" name="PngyuMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>555</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PngyuMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame_speed,
QFrame#frame_dithered,
QFrame#frame_colors,
QFrame#frame_output_filename,
QFrame#frame_ie6
{
    border: 1px solid rgb(127,127,127,63);
    border-radius: 4px;
    padding: 2px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_output_option">
      <property name="title">
       <string>Output Option</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_output_filename">
         <property name="title">
          <string>Output Filename</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="margin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_output_filename">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="margin">
              <number>2</number>
             </property>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineEdit_output_file_prefix">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_outoput_original_name">
               <property name="text">
                <string>+ Original Name +</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLineEdit" name="lineEdit_output_file_suffix">
               <property name="text">
                <string>.png</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_output_option_other">
         <property name="title">
          <string>Other</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <property name="margin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_overwrite">
            <property name="text">
             <string>Overwrite</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_output_directory_mode">
         <property name="title">
          <string>Output Directory</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <property name="margin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_output_same_directory">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Same Directory</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_output_directory_mode</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_output_other_directory">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Other Directory</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_output_directory_mode</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit_output_directory">
            <property name="placeholderText">
             <string>Input output directory or drop it here ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_filelist">
      <property name="title">
       <string>File List</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget_filelist">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_filelist_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton_exec">
      <property name="text">
       <string>exec</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_compress_option">
      <property name="title">
       <string>Compress Option</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_colors">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="margin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_colors">
            <property name="text">
             <string>Colors</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spinBox_colors">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>256</number>
            </property>
            <property name="value">
             <number>256</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSlider_colors">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>7</number>
            </property>
            <property name="value">
             <number>7</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_ie6">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="margin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_ie6_support">
            <property name="text">
             <string>IE6 alpha support</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_dithered">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_dithered">
            <property name="text">
             <string>Dithered</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame_speed">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="margin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_compress_high_quality">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>High Quality</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSlider_compress_speed">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_compress_high_speed">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>High Speed</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_output_directory_mode"/>
 </buttongroups>
</ui>
